test/Maestro/US/NewUserRegistration.yml

```
@@ -10,7 +10,8 @@ env:
 - launchApp:
     clearState: true
 
-- tapOn: "Select country"
+- tapOn:
+    id: "icon-FontAwesomeSolid-62845"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)